Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -25,10 +25,13 @@
 <div id="toolbar">
 <div id="terrainMenu"></div>
 <div id="zoomButtons"></div>
-<div id="toggleLighting"></div>
 <div id="sampleButtons"></div>
 <table><tbody>
     <tr>
+        <td>Toggle Lighting</td>
```

Comment:
Since we're using a checkbox now, I think it would make more sense to say "Enable lighting" here, like we do for the fog checkbox.
And in that case, lighting should be enabled when the checkbox is ticked rather than the other way around (like now).